src/avram/params.cr

```diff
@@ -1,10 +1,12 @@
 class Avram::Params
   include Avram::Paramable
 
-  @hash : Hash(String, Array(String))
+  @hash : Hash(String, Array(String) | String) |  \
+    Hash(String, Array(String)) |  \
+    Hash(String, String)
```

Comment:
oh, interesting! I think this is where I got tripped up before. I didn't think to also do a union of each one. Nice 👍